pkg/controller/mustgather/mustgather_controller.go

```diff
@@ -194,8 +188,8 @@ func (r *ReconcileMustGather) Reconcile(request reconcile.Request) (reconcile.Re
 	// get operator namespace to manage resources in
 	operatorNs, err := k8sutil.GetOperatorNamespace()
 	if err != nil {
-		log.Error(err, "unable to get operator namespace")
-		return r.ManageError(instance, err)
+		operatorNs = "openshift-must-gather-operator"
+		fmt.Println("using default operator namespace: " + operatorNs)
```

Comment:
why shoudn't this be an debug?
```suggestion
		log.V(1).Info("using default operator namespace: " + operatorNs)
```

Reply:
+1